### PR TITLE
THZ Data collection equipment change

### PR DESCRIPTION
**Pull request by Thomas-Benson**
- Replaced Kinemetrics Obsidian (1156) with Kinemetrics Obsidian (617)
- For LC 21 (HNZ, HNN, HNE) streams

### CODE REVIEW

File: install/streams.csv [hunk 1]
```diff
@@ -2128,7 +2128,8 @@ THZ,11,S,H,50,false,false,true,2003-08-08T03:21:00Z,2003-09-24T11:54:00Z
 THZ,20,H,N,200,false,false,true,2003-08-08T00:00:00Z,2014-04-03T13:55:00Z
 THZ,21,B,N,50,false,false,false,2014-04-03T14:05:00Z,2020-06-16T00:00:00Z
 THZ,21,H,N,200,false,false,true,2014-04-03T14:05:00Z,2020-02-08T23:59:59Z
-THZ,21,H,N,200,false,false,false,2020-02-09T00:00:00Z,9999-01-01T00:00:00Z
+THZ,21,H,N,200,false,false,false,2020-02-09T00:00:00Z,2023-05-16T23:55:00Z
+THZ,21,H,N,200,false,false,false,2023-05-22T22:52:00Z,9999-01-01T00:00:00Z
```
Comment (CBurton90) [inline install/streams.csv:2132]: @Thomas-Benson I'm unsure if you have to open a new stream if there haven't been any changes to the instrument configuration. I'd say you can leave the existing line as is with no closing date. I'll let @mnaguit confirm!

First delta PR too, nice one 🎉

Comment (mnaguit) [reply install/streams.csv:2132]: I agree with @CBurton90 . We generally add a new entry for a station when there are changes in streams, like from non-axial to axial and/ triggered to continuous. In this case, THZ remains non-axial & continuous, no updates required. 

Thank you @Thomas-Benson 👍

Comment (Thomas-Benson) [reply install/streams.csv]: Sorry, those changes have been reverted now. Thanks!

**Review by mnaguit — APPROVED** [review]
👍

**Review by CBurton90 — APPROVED** [review]

**Pull request merged**